iaso/api/entity.py

```diff
@@ -233,6 +233,19 @@ def get_queryset(self):
 
         queryset = Entity.objects.filter(account=self.request.user.iaso_profile.account)
 
+        queryset = queryset.prefetch_related(
+            "attributes",
+            "attributes__created_by",
+            "attributes__created_by__teams",
```

Comment:
```suggestion
            "attributes__created_by__teams",
```

`attributes__created_by__teams` already contains all the needed information, so the previous arguments `attributes` and `attributes__created_by` are actually redundant.